packages/cardpay-sdk/sdk/currency-utils/type-conversion.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * This file contains functions that convert BigNumberish to hex, string, and number
+ */
+
+import BigNumber from 'bignumber.js';
+import { BigNumberish } from './types';
+
+/**
+ * Converts hex number to string
+ */
+export const convertHexToString = (hex: BigNumberish): string => new BigNumber(hex).toFixed();
+
+/**
+ * Converts numeric string to hex number
+ */
+export const convertStringToHex = (stringToConvert: string): string => new BigNumber(stringToConvert).toString(16);
+
+/**
+ * Converts a string to a number
+ */
+export const convertStringToNumber = (value: BigNumberish) => new BigNumber(value).toNumber();
+
+/**
+ * Converts `value` to a string. Can return `NaN`.
+ */
+export const convertNumberToString = (value: BigNumberish): string => new BigNumber(value).toFixed();
```

Comment:
Functions in this file are misleadingly named. They really just convert any valid `BigNumberish` to the specified type.